pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -327,78 +328,119 @@ func (k *Bootstrapper) client(ip string, port int) (*kubernetes.Clientset, error
 	}
 
-	if cfg.VerifyComponents[kverify.APIServerWaitKey] {
+	if cc.VerifyComponents[kverify.APIServerWaitKey] {
+		start := time.Now()
 		client, err := k.client(hostname, port)
 		if err != nil {
 			return errors.Wrap(err, "get k8s client")
 		}
-		if err := kverify.WaitForAPIServerProcess(cr, k, cfg, k.c, start, timeout); err != nil {
+		if err := kverify.WaitForAPIServerProcess(cr, k, cc, k.c, start, timeout); err != nil {
 			return errors.Wrap(err, "wait for apiserver proc")
 		}
 
-		if err := kverify.WaitForHealthyAPIServer(cr, k, cfg, k.c, client, start, hostname, port, timeout); err != nil {
+		if err := kverify.WaitForHealthyAPIServer(cr, k, cc, k.c, client, start, hostname, port, timeout); err != nil {
 			return errors.Wrap(err, "wait for healthy API server")
 		}
+		out.T(out.CheckOption, "verifying api server {{.seconds}}", out.V{"seconds": timeToSecond(time.Since(start))})
 	}
 
-	if cfg.VerifyComponents[kverify.SystemPodsWaitKey] {
+	if cc.VerifyComponents[kverify.SystemPodsWaitKey] {
+		start := time.Now()
 		client, err := k.client(hostname, port)
 		if err != nil {
 			return errors.Wrap(err, "get k8s client")
 		}
-		if err := kverify.WaitForSystemPods(cr, k, cfg, k.c, client, start, timeout); err != nil {
+		if err := kverify.WaitForSystemPods(cr, k, cc, k.c, client, start, timeout); err != nil {
 			return errors.Wrap(err, "waiting for system pods")
 		}
+		out.T(out.CheckOption, "verifying system pods {{.seconds}}", out.V{"seconds": timeToSecond(time.Since(start))})
 	}
 
-	if cfg.VerifyComponents[kverify.DefaultSAWaitKey] {
+	if cc.VerifyComponents[kverify.DefaultSAWaitKey] {
+		start := time.Now()
 		client, err := k.client(hostname, port)
 		if err != nil {
 			return errors.Wrap(err, "get k8s client")
 		}
 		if err := kverify.WaitForDefaultSA(client, timeout); err != nil {
 			return errors.Wrap(err, "waiting for default service account")
 		}
+		out.T(out.CheckOption, "verifying default service account {{.seconds}}", out.V{"seconds": timeToSecond(time.Since(start))})
 	}
 
-	if cfg.VerifyComponents[kverify.AppsRunning] {
+	if cc.VerifyComponents[kverify.AppsRunning] {
+		start := time.Now()
 		client, err := k.client(hostname, port)
 		if err != nil {
 			return errors.Wrap(err, "get k8s client")
 		}
 		if err := kverify.WaitForAppsRunning(client, kverify.AppsRunningList, timeout); err != nil {
 			return errors.Wrap(err, "waiting for apps_running")
 		}
+		out.T(out.CheckOption, "verifying apps running {{.seconds}}", out.V{"seconds": timeToSecond(time.Since(start))})
+	}
+
+	if cc.VerifyComponents[kverify.NodeCondition] {
+		start := time.Now()
+		client, err := k.client(hostname, port)
+		if err != nil {
+			return errors.Wrap(err, "get k8s client")
+		}
+		if err := kverify.NodeConditions(client, cc.Driver); err != nil {
+			return errors.Wrap(err, "verifying node conditions")
+		}
+		out.T(out.CheckOption, "verifying node health {{.seconds}}", out.V{"seconds": timeToSecond(time.Since(start))})
```

Comment:
`verifying` implies that this will be done in the future - but this appears to have been done in the past.

I prefer pre-announcing what's going on -- it's not so important once it's over.

Reply:
at first I had it before, but that woudn't have the seconds. any suggestion how to append suggestion to a line once that line is written using out package?